config/config.go

```diff
@@ -1609,8 +1609,12 @@ func (c *Config) getParserConfig(name string, tbl *ast.Table) (*parsers.Config,
 					c.getFieldStringMap(subtbl, "fields_int", &subcfg.FieldsInt)
 					c.getFieldString(subtbl, "field_selection", &subcfg.FieldSelection)
 					c.getFieldBool(subtbl, "field_name_expansion", &subcfg.FieldNameExpand)
-					c.getFieldString(subtbl, "field_name", &subcfg.FieldNameQuery)
-					c.getFieldString(subtbl, "field_value", &subcfg.FieldValueQuery)
+					c.getFieldString(subtbl, "field_name", &subcfg.TagNameQuery)
+					c.getFieldString(subtbl, "field_value", &subcfg.TagValueQuery)
```

Comment:
Are these two really supposed to be `tagnamequery` and `tagvaluequery` and not the corresponding `field` values?

Reply:
Very good catch @powersj! Fixed.